build.gradle

```diff
@@ -1,35 +1,15 @@
-apply plugin: 'com.android.library'
-
-android {
-    compileSdkVersion rootProject.ext.compileSdkVersion
-    buildToolsVersion rootProject.ext.buildToolsVersion
-
-    defaultConfig {
-        minSdkVersion 10
+buildscript {
+    repositories {
+        jcenter()
     }
 
-    packagingOptions {
-        exclude 'META-INF/LICENSE'
-        exclude 'META-INF/LICENSE.txt'
-        exclude 'META-INF/NOTICE'
-        exclude 'META-INF/NOTICE.txt'
-        exclude 'LICENSE.txt'
-    }
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_7
-        targetCompatibility JavaVersion.VERSION_1_7
-    }
-    lintOptions {
-        disable 'InvalidPackage'
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.3.1'
```

Comment:
Also, how did we go from a module-style build.gradle to a project-style build.gradle?